spec/spec.md

```diff
@@ -310,29 +315,31 @@ where:
 - `dapp_uri`: (Optional) a URI representing the web address associated with the dapp endpoint making this authorization request. If present, it must be an absolute, hierarchical URI.
 - `dapp_icon_relative_path`: (Optional) a relative path (from dapp_uri) to an image asset file of an icon identifying the dapp endpoint making this authorization request
 - `dapp_name`: (Optional) the display name for this dapp endpoint
+- `cluster`: (optional) if set, the Solana network cluster with which the dapp endpoint intends to interact; supported values include `mainnet_beta`, `testnet`, `devnet`. If not set, the wallet endpoint determines which cluster to send transactions to.
```

Comment:
```suggestion
- `cluster`: (optional) if set, the Solana network cluster with which the dapp endpoint intends to interact; supported values include `mainnet-beta`, `testnet`, `devnet`. If not set, the wallet endpoint determines which cluster to send transactions to.
```

To keep consistent with the cluster type here:

https://github.com/solana-labs/solana/blob/ef30f1729c91e46833247f37173655cfded7e02f/web3.js/src/util/cluster.ts#L14

Reply:
Done